docs/developer/README.md

```diff
@@ -34,4 +34,20 @@ You can change the logging verbosity, if needed. Do a web search for "django log
 
 ## Running tests
 
-tbd.
+Django's test suite:
```

Comment:
Add a note to make sure Docker is running and that this has to be run from `src`?

Reply:
Added!